README.md

```diff
@@ -490,6 +491,24 @@ DESCRIPTION
 
 _See code: [src/commands/logout.js](https://github.com/apify/apify-cli/blob/v0.16.2/src/commands/logout.js)_
 
+## `apify pull [ACTORID]`
+
+Pulls the latest version of an Actor from the Apify platform to the current directory.
```

Comment:
What does it mean the "latest" ? IMHO by default it should pull the version with the default build tag, which is typically `version-0`

Reply:
Please also add explanation what will happen for Web IDE and Git repo types, so that users don't need to guess

Reply:
And what happens on push after? Will it overwrite what's in Web IDE? If so, we should warn users about it

Reply:
Warning for push command is already there: https://github.com/apify/apify-cli/blob/master/src/commands/push.js#L193